<!DOCTYPE html>
<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>

    <link href="/favicon.ico" rel="shortcut icon" />

    <title>File libmng-1.0.10-mingw.patch of Package mingw32-libmng - openSUSE Build Service</title>

    <link href="/assets/application-5d4d169f6dc1991ea2edadaac45cceee.css" media="screen" rel="stylesheet" type="text/css" />
      <style type="text/css">
	.CodeMirror { border-width: 0 0 0 0; }
.CodeMirror-scroll {
height: 660px;
width: auto; 
}

      </style>
    <script src="/assets/cm2-9639766719eb3568a0f5fbae1647f2cd.js" type="text/javascript"></script>

    <script src="/assets/application-566fa119bb93469819c0cfcbb2bebe58.js" type="text/javascript"></script>
    <script type="text/javascript">
//<![CDATA[

      
      var _paq = _paq || [];
      $(function() {
        
  $("#advanced_tabs_trigger").click(function() {
    $("#advanced_tabs").show();
    $("#advanced_list_item").hide();
  });
  use_codemirror(0, true, 'diff');

          callPiwik();  
      });

//]]>
</script>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="rIuva2e702ohmC63+2ViFQ0iHt2idp8H3AuKZMRKmEI=" name="csrf-token" />
  </head>
  <body>
        <!-- Start: Header -->
    <div id="header">

      <div id="header-content" class="container_12">

        <a href="/" id="header-logo"><span class="icons-header-logo inlineblock"></span></a>
      <ul id="global-navigation">
        <li id="item-downloads"><a href="http://en.opensuse.org/openSUSE:Browse#Downloads">Downloads</a></li>
        <li id="item-support"><a href="http://en.opensuse.org/openSUSE:Browse#Support">Support</a></li>
        <li id="item-community"><a href="http://en.opensuse.org/openSUSE:Browse#Community">Community</a></li>
        <li id="item-development"><a href="http://en.opensuse.org/openSUSE:Browse#Development">Development</a></li>
        </ul>

        
  
  
<form accept-charset="UTF-8" action="/search" class="label-overlay" id="global-search-form" method="post"><div style="margin:0;padding:0;display:inline"><input name="utf8" type="hidden" value="&#x2713;" /><input name="authenticity_token" type="hidden" value="rIuva2e702ohmC63+2ViFQ0iHt2idp8H3AuKZMRKmEI=" /></div>  <div style="display: inline">
    <label for="search" class="hidden">Search</label>
    <input id="search" name="search_text" type="text" value="" />    <input type="submit" value="Search" class="hidden" />
  </div>
</form>


      </div>
    </div>
    <!-- End: Header -->



    <div id="subheader" class="container_16">
      <div id="breadcrump" class="grid_10 alpha">
        <span class="icons-home_grey inlineblock"></span><a href="/">openSUSE Build Service</a>
            &gt;
                          <a href="/project/list_public">Projects</a>
            &gt;
             <!-- multiple items in crump list but they belong together as sub group, eg sub projects -->
              <a href="/project/show?project=windows%3Amingw">windows:mingw</a>:<a href="/project/show?project=windows%3Amingw%3Awin32">win32</a>
            &gt;
                          <a href="/package/show?package=mingw32-libmng&amp;project=windows%3Amingw%3Awin32">mingw32-libmng</a>
            &gt;
                          libmng-1.0.10-mingw.patch
      </div>

      <div class="grid_6 omega" style="text-align: right;">
        <a href="https://secure-www.novell.com/selfreg/jsp/createOpenSuseAccount.jsp?%22https://build.opensuse.org/package/view_file?expand=1&amp;file=libmng-1.0.10-mingw.patch&amp;package=mingw32-libmng&amp;project=windows%3Amingw%3Awin32%22">Register</a> |
      <a href="/user/login" id="login-trigger">Login</a>
      <div id="login-form">
        <form accept-charset="UTF-8" action="https://build.opensuse.org/ICSLogin/auth-up" enctype="application/x-www-form-urlencoded" id="login_form" method="post"><div style="margin:0;padding:0;display:inline"><input name="utf8" type="hidden" value="&#x2713;" /><input name="authenticity_token" type="hidden" value="rIuva2e702ohmC63+2ViFQ0iHt2idp8H3AuKZMRKmEI=" /></div>
          <p>
            <input id="url" name="url" type="hidden" value="https://build.opensuse.org/package/view_file?expand=1&amp;file=libmng-1.0.10-mingw.patch&amp;package=mingw32-libmng&amp;project=windows%3Amingw%3Awin32" />
            <input id="context" name="context" type="hidden" value="default" />
            <input id="proxypath" name="proxypath" type="hidden" value="reserve" />
            <input id="message" name="message" type="hidden" value="Please log in" />
            <input id="return_to_path" name="return_to_path" type="hidden" value="/package/view_file?expand=1&amp;file=libmng-1.0.10-mingw.patch&amp;package=mingw32-libmng&amp;project=windows%3Amingw%3Awin32" />
            <label for="username">Username</label>
            <input id="username" name="username" type="text" value="" />
          </p>
          <p>
            <label for="password">Password</label>
            <input id="password" name="password" type="password" value="" />
          </p>
          <p><input name="commit" onclick="fillEmptyFields();" type="submit" value="Login" /></p>
          <p class="slim-footer"><a href="#" id="close-login">Cancel</a></p>
</form>      </div>
</div>

    </div>

    


    <!-- this is needed for the delete confirm dialogues -->
    <div id="dialog_wrapper" style="display: none"></div>

    <!-- Start: Main Content Area -->
    <div id="content" class="container_16 content-wrapper">
        <div class="grid_16 box box-shadow alpha omega">
          

<div class="box-header header-tabs" id="package_tabs">
  <ul>
    <li id="tab-overview"><a href="/package/show?package=mingw32-libmng&amp;project=windows%3Amingw%3Awin32">Overview</a></li>
      <li id="tab-repositories"><a href="/package/repositories?package=mingw32-libmng&amp;project=windows%3Amingw%3Awin32">Repositories</a></li>
      <li id="tab-revisions"><a href="/package/revisions?package=mingw32-libmng&amp;project=windows%3Amingw%3Awin32">Revisions</a></li>
      <li id="tab-requests"><a href="/package/requests?package=mingw32-libmng&amp;project=windows%3Amingw%3Awin32">Requests</a></li>
      <li id="tab-users"><a href="/package/users?package=mingw32-libmng&amp;project=windows%3Amingw%3Awin32">Users</a></li>
      <li id="advanced_list_item"><a href="#" id="advanced_tabs_trigger">Advanced</a></li>
  </ul>
    <div id="advanced_tabs" class="hidden">
      <ul>
        <li id="tab-attributes"><a href="/package/attributes?package=mingw32-libmng&amp;project=windows%3Amingw%3Awin32">Attributes</a></li>
        <li id="tab-meta"><a href="/package/meta?package=mingw32-libmng&amp;project=windows%3Amingw%3Awin32">Meta</a></li>
      </ul>
    </div>
</div>


<h3>File libmng-1.0.10-mingw.patch of Package mingw32-libmng </h3>

    


<textarea class="editor" cols="0" id="editor_0" rows="0" style="display:
	  inline-block">
diff -u -r libmng-1.0.10/makefiles/makefile.mingwdll libmng-1.0.10/makefiles/makefile.mingwdll
--- libmng-1.0.10/makefiles/makefile.mingwdll	2005-01-30 11:28:00.000000000 +0100
+++ libmng-1.0.10/makefiles/makefile.mingwdll	2009-12-08 09:35:23.000000000 +0100
@@ -10,8 +10,8 @@
 #
 
 # outputs
-LIBMNG_A   = libmng.a
-LIBMNG_DLL   = libmng.1.dll
+LIBMNG_A   = libmng.dll.a
+LIBMNG_DLL   = libmng-1.dll
 INSTALL_PREFIX = C:/MinGW/
 # maybe you sould replace with anti-slashes
 
@@ -51,7 +51,7 @@
 # for i386:
 #ALIGN=-malign-loops=2 -malign-functions=2
 
-CFLAGS=$(ZLIBINC) $(JPEGINC) $(LCMSINC) -Wall -O3 -funroll-loops $(OPTIONS) $(ALIGN) -s
+CFLAGS+=$(ZLIBINC) $(JPEGINC) $(OPTIONS) $(ALIGN) -s
 LDFLAGS=-L. -lmng $(ZLIBLIB) $(JPEGLIB) $(LCMSLIB) -lm -s
 
 INCPATH=$(prefix)/include
@@ -88,9 +88,11 @@
 $(LIBMNG_A): $(LIBMNG_DLL)
 
 $(LIBMNG_DLL) : $(OBJS)
-	dllwrap --implib=$(LIBMNG_A) --dllname=$(LIBMNG_DLL) $(OBJS) $(LDFLAGS)
+	$(CC) --shared -o $(LIBMNG_DLL) $(OBJS) -Wl,--out-implib,$(LIBMNG_A) -Wl,--kill-at $(JPEGLIB) $(ZLIBLIB)
    
 install : $(LIBMNG_A)
+	$(MKDIR)                 $(INSTALL_PREFIX)bin
+	$(COPY)  $(LIBMNG_DLL)   $(INSTALL_PREFIX)bin
 	$(MKDIR)                 $(INSTALL_PREFIX)include	
 	$(COPY)  libmng.h        $(INSTALL_PREFIX)include
 	$(COPY)  libmng_conf.h   $(INSTALL_PREFIX)include
@@ -99,7 +101,7 @@
 	$(COPY)  $(LIBMNG_A)     $(INSTALL_PREFIX)lib
 
 clean:
-	$(RM) *.o
+	$(RM) *.o $(LIBMNG_DLL) $(LIBMNG_A)
 
 # DO NOT DELETE THIS LINE -- make depend depends on it.
 
</textarea>




        </div>
    </div>
    <!-- End: Main Content Area -->

    <div style="clear: both;"></div>
    <div id="footer" class="container_12">

      <div class="box_content grid_3">
        <strong class="grey-medium spacer1">Locations</strong>
        <ul>
          <li><a href="/project/list_public">Projects</a></li>
          <li><a href="/search" class="search-link">Search</a></li>
            <li><a href="/monitor">Status Monitor</a></li>
        </ul>
      </div>


      <div class="box_content grid_3">
        <strong class="grey-medium spacer1">Help</strong>
        <ul>
              <li><a href="http://en.opensuse.org/Portal:Build_Service">Open Build Service Portal</a></li>
              <li><a href="http://en.opensuse.org/openSUSE:Build_Service_Tutorial">Building Packages</a></li>
              <li><a href="http://en.opensuse.org/openSUSE:Specfile_guidelines">Writing spec Files</a></li>
              <li><a href="http://en.opensuse.org/openSUSE:Submitting_bug_reports">Reporting a Bug</a></li>
              <li><a href="http://forums.opensuse.org/english/other-forums/development/open-build-service-obs/">Forums</a></li>
        </ul>
      </div>

      

  <strong class="grey-medium spacer1">Sponsor</strong>
  <p>
    <a href="http://en.opensuse.org/Sponsors"><span class="icons-sponsor_heinlein inlineblock"></span></a>
  </p>


      <div id="footer-legal" class="grid_12">
        <p>
          <a href="http://openbuildservice.org">Open Build Service (OBS)</a> is an <a href="http://www.opensuse.org">openSUSE project</a>.
        </p>
      </div>
    </div>
  </body>
</html>
